src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user2 = new User();
+    private static User user3 = new User();
+    private static RegistrationServiceImpl registrationService;
+    private User defaultUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        String loginBruce = "Bruce";
+        int validUser2Age = 100;
+        String reverseGoodPassword = "654321";
+        user2.setLogin(loginBruce);
+        user2.setAge(validUser2Age);
+        user2.setPassword(reverseGoodPassword);
+        String loginAmy = "Amy";
+        int validEighteenAge = 18;
+        String goodPassword = "parol2";
+        user3.setLogin(loginAmy);
+        user3.setAge(validEighteenAge);
+        user3.setPassword(goodPassword);
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        int validDefaultAge = 18;
+        String loginDefault = "Kate";
+        defaultUser = new User();
+        defaultUser.setLogin(loginDefault);
+        defaultUser.setAge(validDefaultAge);
+        String goodPassword = "123456";
+        defaultUser.setPassword(goodPassword);
+    }
+
+    @Test
+    void addUser_SameLogin_NotOk() {
```

Comment:
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.
```suggestion
    void register_sameLogin_NotOk() {
```